resources/lib/WatchlistFlavor/__init__.py

```diff
@@ -30,6 +32,20 @@ def get_active_flavor():
 
         return WatchlistFlavor.__SELECTED
 
+    @staticmethod
+    def check_token_expiration():
+        login_ts = control.getSetting(WatchlistFlavor.__LOGIN_TS_KEY)
+        if not login_ts:
+            return True
+
+        expires_in = 2591963 # Seconds until the access_token expires (30 days)
```

Comment:
I meant globally with Capital letters, but this also works, original version was very amature..

Reply:
Want to merge as is or make it global? (Like login keys for example)

Reply:
oh ok, just a sec will make it global

Reply:
Thanks and sorry for the trouble

Reply:
@DxCx  actually just merge as is

Reply:
my bad